pkg/clusterprovider/providers/kstone/kstone.go

```diff
@@ -46,11 +46,13 @@ const (
 	AnnoImportedURI = "importedAddr"
 )
 
+// EtcdClusterKstone is the etcd cluster created from kstone-dashboard
```

Comment:
// EtcdClusterKstone is responsible for synchronizing kstone.tkestack.io/etcdcluster to kstone-etcd-operator which will be responsible for the real implementation of the etcd cluster.

Reply:
done